libraries/bezier-rs/src/subpath/transform.rs

```diff
@@ -75,13 +86,184 @@ impl Subpath {
 			(Subpath::new(first_split, false), Some(Subpath::new(second_split, false)))
 		}
 	}
+
+	/// Returns [ManipulatorGroup]s with a reversed winding order.
+	fn reverse_manipulator_groups(manipulator_groups: &[ManipulatorGroup]) -> Vec<ManipulatorGroup> {
+		manipulator_groups
+			.iter()
+			.rev()
+			.map(|group| ManipulatorGroup {
+				anchor: group.anchor,
+				in_handle: group.out_handle,
+				out_handle: group.in_handle,
+			})
+			.collect::<Vec<ManipulatorGroup>>()
+	}
+
+	/// Returns a [Subpath] with a reversed winding order.
+	pub fn reverse(&self) -> Subpath {
+		Subpath {
+			manipulator_groups: Subpath::reverse_manipulator_groups(&self.manipulator_groups),
+			closed: self.closed,
+		}
+	}
+
+	/// Returns an open [Subpath] that results from trimming the original Subpath between the points corresponding to `t1` and `t2`.
+	/// If `t1` > `t2`, then behavior for computing the resulting Subpath will differ depending on whether the original Subpath is open or closed:
```

Comment:
I'm thinking this should be kept consistent to avoid confusion, so the Subpath returned by this function will always have the same winding order as the original. That means, in the case when `t1` > `t2`, swapping `t1` and `t2` so it behaves the same as if the user called this function with the arguments reversed. Then change the last line of these doc comments to say that, more generally, if the reverse direction is desired from the winding order of the original curve, to call `Subpath::reverse`.

(Please also read over this function's doc comments, which I updated for clarity, to confirm my claims are accurate.)

Reply:
The current behavior for the open subpath case is consistent with the trim behavior for a `Bezier`. Should we change it to match what you're suggesting here as well?

Reply:
I think that's wise. In both a segment and a subpath, treat it agnostic of argument order. People can call reverse if they want it reversed. This results in the least unexpected behavior.

Reply:
Just to confirm: For a open subpath + a single segment, we perform the trim agnostic of argument order. Does this apply for a closed subpath as well? Or do we return the curve that crosses the break when `t1 > t2`?

Reply:
We need to provide some mechanism to trim closed subpaths across the break. I think the current implementation for closed subpaths respects the 'winding' order that @Keavon described

Reply:
Oh yes, argument order would matter for closed subpaths so it decides which way to go around the closed shapes. But in all cases, it would follow the existing winding order, that's what I meant when saying it was agnostic of argument order (which is an accurate description for bezier segments, and for open subpaths).